example/lib/pages/simple/image_editor_demo.dart

```diff
@@ -26,14 +26,31 @@ class _SimpleImageEditorState extends State<SimpleImageEditor> {
   final GlobalKey<ExtendedImageEditorState> editorKey =
       GlobalKey<ExtendedImageEditorState>();
   bool _cropping = false;
+  String imageSource = 'assets/example.gif';
```

Comment:
don't change this demo, we don't need more image asset, it's just a demo. More source mean the customer download more slowly. By the way It seems you don't add example.gif into demo, double check if you want to pull a pr.